themes/default/content/blog/docker-build/index.md

```diff
@@ -0,0 +1,744 @@
+This addition builds on the solid foundation of our widely utilized [Pulumi Docker] provider.
+By embracing the power of Docker's [BuildKit] technology, this provider enhances your building experience, allowing you to modernize your image builds.
+This launch signifies our ongoing commitment to advancing technology and empowering developers to achieve new heights in container management.
+
+The Docker-Build provider is crafted to optimize the efficiency and versatility of Docker builds and offers developers a more potent and flexible approach to building Docker images, ensuring that they can achieve high performance and scalability in their deployment workflows.
+The new provider exposes the full power of Docker's next-gen [buildx](https://docs.docker.com/reference/cli/docker/buildx/build/) functionality and includes key features such as,
+
+* __Multi-platform image support__: Build images that run seamlessly across different hardware architectures.
+
+* __Advanced caching mechanisms__: Utilize various caching strategies to speed up builds and reduce resource consumption.
+
+* __Support for secrets and Pulumi [ESC](../environments-secrets-configurations-management) (Environments, Secrets and Configuration)__: Safely manage sensitive information and configuration with first-class support, integrating seamlessly into your deployment workflows.
+
+* __Support for multiple export types__: Increase the power and flexibility of your workflows by exporting your images to registries, disk, or blob storage.
+
+* __Support for Docker Build Cloud__: Take advantage of [Docker Build Cloud](https://www.docker.com/products/build-cloud/) to scale your build processes effortlessly and manage build resources dynamically, enhancing productivity and performance.
+
+For background, the [Pulumi Docker] provider was first introduced in 2018 and
+has seen wide adoption.
+Over the years Pulumi has continued our investment in it, notably introducing a
+version 4.0 [last year](../build-images-50x-faster-docker-v4) and additional
+improvements like build-on-preview behavior after that.
+
+At the same time, the Docker build ecosystem has seen tremendous advancements
+in how images can be built and distributed. These changes were initially
+experimental, but they became official when [BuildKit] graduated to become the
+default builder in Docker version 23.
+
+We have heard a large number of requests from users to expose more [BuildKit]
+functionality in the Pulumi Docker provider. After some consideration, we
+decided that a new, standalone provider, focused exclusively on building
+images, would provide the best overall user experience as well as the best
+platform for Pulumi to stay current with the evolving build landscape.
+
+## Getting started
+
+{{< chooser language "typescript,python,csharp,go,yaml,java" / >}}
+
+Add the "docker-build" package to your project to start taking advantage of modern image builds with Pulumi.
```

Comment:
Instead of this code snippet that just shows the import statement, is it worth having a full example of the simplest possible use case?  Like https://www.pulumi.com/registry/packages/docker/api-docs/image/#a-docker-image-build-and-push.

This also helps given that the next example is quite complex (which is nice, but feels like we're skipping a step of showing the basic use case of just this new provider/resource).